src/applications/gqrx/remote_control.cpp

```diff
@@ -366,26 +366,73 @@ void RemoteControl::startRead()
      */
     else if (cmdlist[0] == "\\dump_state")
     {
-        rc_socket->write("0\n"
+        rc_socket->write(
+                         /* rigctl protocol version */
+                         "0\n"
+                         /* rigctl model */
                          "2\n"
+                         /* ITU region */
                          "1\n"
-                         "150000.000000 30000000.000000  0x900af -1 -1 0x10000003 0x3\n"
+                         /* RX/TX frequency ranges
+                          * start, end, modes, low_power, high_power, vfo, ant
+                          *  start/end - Start/End frequency [Hz]
+                          *  modes - Bit field of RIG_MODE's (AM|CW|CWR|USB|LSB|FM|WFM)
+                          *  low_power/high_power - Lower/Higher RF power in mW,
+                          *                         -1 for no power (ie. rx list)
+                          *  vfo - VFO list equipped with this range (RIG_VFO_MAIN)
+                          *  ant - Antenna list equipped with this range, 0 means all
+                          *  FIXME: limits can be gets from receiver::get_rf_range()
+                          */
+                         "1.000000 1000000000.000000 0xef -1 -1 0x4000000 0x0\n"
```

Comment:
I am aware of these issues but currently we do not have the framework to use the correct parameters depending on which device is being used. We can add it later if it is really absolutely necessary, but I want to keep this practical and only add thins that are necessary to make applications work.